codeship-services.yml

```diff
@@ -0,0 +1,111 @@
+version: '2'
+
+services:
+  
+  mfaidp:
+    image: silintl/ssp-base:feature_mfa
```

Comment:
Once we release this as `1.0.0`, will we then update ssp-base to use that version, then update this MFA-module again so that they use `ssp-base:develop` (and/or `ssp-base:master`)?

Reply:
yeah I suppose so, develop I guess?